spynnaker/pyNN/models/neuron/synaptic_matrices.py

```diff
@@ -13,56 +13,62 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import math
 import numpy
-from collections import defaultdict
+from collections import namedtuple
 
-from spinn_utilities.ordered_set import OrderedSet
 from pacman.model.routing_info import BaseKeyAndMask
 from data_specification.enums.data_type import DataType
 
 from spinn_front_end_common.utilities.constants import BYTES_PER_WORD
 from spynnaker.pyNN.data import SpynnakerDataView
 from spynnaker.pyNN.models.neuron.master_pop_table import (
     MasterPopTableAsBinarySearch)
-from spynnaker.pyNN.utilities.utility_calls import get_n_bits
 from spynnaker.pyNN.utilities.constants import SPIKE_PARTITION_ID
-from .key_space_tracker import KeySpaceTracker
```

Comment:
This is the only use off KeySpaceTracker which is the only use of ElementAllocatorAlgorithm which is the only user of ElementFreeSpace

So why keep these?

Reply:
Removed